mailbox/store/src/test/java/org/apache/james/mailbox/store/quota/ListeningCurrentQuotaUpdaterTest.java

```diff
@@ -105,4 +107,15 @@ public void emptyAddedEventShouldNotTriggerDecrease() throws Exception {
         verify(mockedCurrentQuotaManager, never()).increase(QUOTA_ROOT, 0, 0);
     }
 
+    @Test
+    public void mailboxDeletionEventShouldDecreaseCurrentQuotaValues() throws Exception {
+        MailboxListener.MailboxDeletion deletion = mock(MailboxListener.MailboxDeletion.class);
```

Comment:
Can't we use a real impl?

Reply:
It's not the idea of this test class which is such to assert that events are fired, whatever the events implementation is.